client/src/components/ColorList.js

```diff
@@ -1,40 +1,70 @@
-import React, { useState } from "react";
-import axios from "axios";
+import React, { useState, useEffect } from "react";
+import axiosWithAuth from "../utils/axiosWithAuth";
 
 const initialColor = {
   color: "",
   code: { hex: "" }
 };
-
+const token = localStorage.getItem(token);
```

Comment:
Dont forget - anytime we're dealing with localStorage, our values should be strings like so:
```
const token = localStorage.getItem("token")
```